webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js

```diff
@@ -221,14 +218,29 @@ class SearchParamsStore {
      */
     static convertQueryToSearchParams(query) {
 
-        let { repoIds, bcp47Tags,
+        let {
             searchAttribute, searchText, searchType,
             status, used, unUsed,
             pageSize, currentPageNumber, pageOffset } = query;
-
+        
+        let repoIds = query["repoIds[]"];
+        let bcp47Tags = query["bcp47Tags[]"];
+        
+        if (typeof repoIds !== "undefined") {
+            if (Array.isArray(repoIds)) {
+                repoIds = repoIds.map((value) => parseInt(value));
+            } else {
+                repoIds = [parseInt(repoIds)];
+            }
+        }
+        
+        if (typeof bcp47Tags !== "undefined"&& ! Array.isArray(bcp47Tags)) {
```

Comment:
ha. i meant to write

`npm install --save-dev prettier` 

it's pretty cool. auto formats so we don't have to .